### ISSUE TITLE
Add CI pipeline

### ISSUE THREAD

**Issue by TheCodez** [TheCodez/dynamic-occupancy-grid-map#31]

**Comment by TheCodez** [comment]
@cbachhuber not including the demo project in the build links successfully. So the problem should stem from the demo project.

**Comment by cbachhuber** [comment]
Right, I already found that as well. I'm quite sure, however, that not the demo project is the problem, but that we are linking everything together to an executable. So at this time, 'undefined reference' errors will surface. 

I had the suspicion that CI does not find the GLEW library, and I'm right:
- I added printing of variable `GLEW_LIBRARY` in [this commit](https://github.com/cbachhuber/dynamic-occupancy-grid-map/commit/92ceca83a7dc9ec30136a62c084f97cf251b0e1d).
- On my machine, I see during configuration
    ```
    -- GLEW LIBRARY LOCATION: 
    -- /usr/lib/x86_64-linux-gnu/libGLEW.so
    ```
- [On CI, I see](https://github.com/cbachhuber/dynamic-occupancy-grid-map/runs/596220080#step:7:40)
    ```
     -- GLEW LIBRARY LOCATION: 
     --
    ```

So I'm currently looking into ways of enabling CI to find GLEW. Do you think that makes sense?

**Comment by cbachhuber** [comment]
Fixed! See [this build](https://github.com/cbachhuber/dynamic-occupancy-grid-map/runs/596279413) :)

**Comment by cbachhuber** [comment]
[This answer](https://stackoverflow.com/a/27487567/7260972) writes `GLEW_LIBRARIES` instead of `GLEW_LIBRARY`. Also [cmake documentation](https://cmake.org/cmake/help/latest/module/FindGLEW.html) promotes `GLEW_LIBRARIES`. So I used that, and it works :)

PR incoming ;)

**Comment by TheCodez** [comment]
Thanks you for testing this. I just pushed a commit to see if `GLEW_LIBRARIES` is the only change needed.

**Comment by cbachhuber** [comment]
It seems that something else is needed. I just created a new branch, but with only the `GLEW_LIBRARIES` change it does not compile. I suggest that I clean up branch [fix-ci](https://github.com/cbachhuber/dynamic-occupancy-grid-map/tree/fix-ci) and create a PR. Ok?

**Comment by TheCodez** [comment]
Sure go ahead 👍 . I suggest rebasing against my master branch as I have removed the build folder. So it has to be created again in the CI.
